cs/ccxt/ws/OrderBook.cs

```diff
@@ -260,6 +261,35 @@ public IOrderBook limit()
     }
 
 
+
+
+    public void reset(object snapshot = null)
+    {
+        lock (_syncRoot)
```

Comment:
we can keep it for now, but I think we can find a better way without repeating the reset implementation (with the interfaces)